Headers/DebugServer2/Target/Darwin/MachOProcess.h

```diff
@@ -21,6 +22,7 @@ class MachOProcess : public ds2::Target::POSIX::Process {
 protected:
   std::string _auxiliaryVector;
   Address _sharedLibraryInfoAddress;
+  Host::Darwin::Mach _mach;
```

Comment:
It's here because it's at the lower level in the hierarchy, since we need it in MachOProcess.cpp. When you delete these file, you can move it up.